codingTracker.jakubmikolajewski/CodingTrackerLibrary/CodingController.cs

```diff
@@ -0,0 +1,279 @@
+﻿using Dapper;
+
+namespace CodingTrackerLibrary
+{   
+    public class CodingController
+    {
+        public static Dictionary<string, string> valuesToUpdate = [];
+        public static void SelectAllSessionsQuery()
+        {        
+            string command = $"SELECT * FROM 'Coding sessions';";
+
+            DatabaseActions current = new DatabaseActions();
+            using (current.Connection)
```

Comment:
I am not exactly sure on your thinking here. But the initialisation of a DatabaseActions object is redundant and makes the code harder to read. Might even stop what you are trying to achieve with the using statement.

I would have just kept is simple with:
`using var connection = new SqlLiteConnection(connectionString);`